app/RouteHandlers/AddDonationHandler.php

```diff
@@ -183,4 +186,33 @@ private function isSubmissionAllowed( Request $request ) {
 		return true;
 	}
 
+	private function collectPiwikTrackingEvents( array $sessionTrackingData, Donation $donation ) {
+		$piwikTracking = new PiwikCustomTracking();
+		if ( isset( $sessionTrackingData['paymentType'] ) ) {
+			$piwikTracking->setCustomVariable(
+				1,
```

Comment:
I was thinking the same thing when I reviewed this. The IDs and their names are defined in Piwik. What the ID means is expressed in the next parameter, the name of the variable so it would probably be silly to have

```
$piwikTracking->setCustomVariable(
    self::VARIABLE_ID_AMOUNT,
    'Amount'
```

The meaning (and the IDs) of the variables is unlikely to change in the future.